build/go-runner/README.md

```diff
@@ -0,0 +1,30 @@
+# Kubernetes go-runner image
```

Comment:
Remove this file.
The go-runner image building is now described here: https://github.com/kubernetes/release/blob/master/images/build/go-runner/README.md